### PR TITLE
Update Documentation

### LINKED ISSUE #2: Update Documentation
- Update User Guides
- Update Developer Guides
- Update README and AboutMe documents

### PR DESCRIPTION
**Pull request by weixue123**
- Add Wei Xue's profile to about-us page
- Update user guide
- Update developer guide
- Add user-interface mock-up

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -126,8 +143,13 @@ Format: `exit`
...
 --------|------------------
+**Add Order** | `addorder t/CHEESE_TYPE q/QUANTITY p/PHONE_NUMBER` <br> e.g. `addorder t/Parmesan q/2 p/65555555`
+**Add Cheese** | `addcheese t/CHEESE_TYPE q/QUANTITY` <br> e.g. `addcheese t/CHEESE_TYPE q/QUANTITY`
+**Add Customer** | `addcustomer n/CUSTOMER_NAME p/PHONE_NUMBER a/ADDRESS` <br> e.g. `addcustomer n/John Doe p/65555555 a/Blk 436 Serangoon Gardens St 26 #01-01`
+**Remove Order** | `removeorder ORDER_INDEX` <br> e.g. `removeorder 2`
+**List Orders** | `listorder`
+**List Cheese** | `listcheese [t/CHEESE_TYPE]` <br> e.g. `listcheese t/Mozzarella`
 **List Customer** | `listcustomer`
-**Mark Done** | `done INDEX`<br> e.g., `done 2`
+**Mark As Done** | `done INDEX` <br> e.g., `done 2`
```
Comment (daniellau88) [inline docs/UserGuide.md]: I think would be nice to remove the comma to standardize

File: docs/AboutUs.md [hunk 2]
```diff
@@ -51,10 +51,10 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
```
Comment (daniellau88) [inline docs/AboutUs.md:49]: Additionally, can you help to remove Line 8 as well (I think not relevant)

**Review by daniellau88 — COMMENTED** [review]
Other than that, the rest looks good :)

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,7 +3,10 @@ layout: page
 title: User Guide
 ---
 
-Chesse Inventory Management (CHIM) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Chesse Inventory Management (CHIM) is a **desktop app for managing the contacts, inventory, and orders of a home-based
+cheese making business. The app is optimized for use via a Command Line Interface** (CLI) while still having the
```
Comment (daniellau88) [inline docs/UserGuide.md]: Sozz, i realized the bolding looks a bit weird, would be good to keep the bolding within the same sentence.

Comment (weixue123) [reply docs/UserGuide.md]: Okay i just removed the bolding entirely, thought it was unnecessary (dk why I typed it this way, must have missed it).

**Review by daniellau88 — APPROVED** [review]
LGTM

**Pull request merged**